_format/1.1/index.md

```diff
@@ -566,6 +601,38 @@ implements at least version 1.0 of the specification.
 > Note: Because JSON API is committed to making additive changes only, the
 version string primarily indicates which new features a server may support.
 
+### <a href="#document-identities-array" id="document-identities-array" class="headerlink"></a> Identities Array
+
+A JSON API document **MAY** include an array of resource identifiers
+under a top level `identities` member. If present, the value of the `identities`
+member **MUST** be an array containing only
+[resource identifier objects][resource identifier object].
+
+The purpose of the `identities` array is to correlate `lid`s in a request with
+`id`s in a response.
```

Comment:
Like @gabesullice, I'm a little fuzzy on why `identities` is strictly necessary - I think you could construct the same information by crawling the `ops` list. But, maybe its purpose is just to simplify that task? I think that would be valid, but it might be nice to have phrasing to that effect. Maybe:
> The purpose of the `identities` array is to make it simple to correlate `lid`s in a request with `id`s in a response.